Q/Qemu/build_tarballs.jl

```diff
@@ -52,9 +91,9 @@ dependencies = [
     Dependency("Glib_jll", v"2.59.0"; compat="2.59.0"),
     Dependency("PCRE_jll"),
     # TOOD: verify Gettext is actually needed at runtime
-    Dependency("Gettext_jll", v"0.20.1"; compat="=0.20.1"),
+    Dependency("Gettext_jll"),
```

Comment:
Gettext always need full compat specification: #3024
```suggestion
    Dependency("Gettext_jll"; compat="=0.21.0"),
```

Reply:
Let's try it:
```suggestion
    BuildDependency("Gettext_jll"),
```

Reply:
Uhm, this was the kind of test I wanted to do:
```console
% for file in x86_64-linux-gnu-cxx11/bin/*; do readelf -d ${file} | grep NEEDED; done | cut -d':' -f 2 | sort -u
 [libc.so.6]
 [libgcc_s.so.1]
 [libgio-2.0.so.0]
 [libglib-2.0.so.0]
 [libgmodule-2.0.so.0]
 [libgobject-2.0.so.0]
 [libm.so.6]
 [libpixman-1.so.0]
 [libpthread.so.0]
 [librt.so.1]
 [libstdc++.so.6]
 [libutil.so.1]
 [libz.so.1]
```
No gettext libraries appear to be needed by the generated executables.

Reply:
Ah, sorry thanks for checking! I did actually test the binaries without the dependency from the job and they seemed to work